scripts/esbuild-octokit-plugin.js

```diff
@@ -0,0 +1,100 @@
+const fs = require("fs/promises");
+const glob = require("glob");
+const esbuild = require("esbuild");
+
+const octokitBundlesPlugin = (options = {}) => {
+  const sharedOptions = {
+    sourcemap: "external",
+    sourcesContent: true,
+    minify: false,
+    allowOverwrite: true,
+    packages: "external",
+    ...options,
+  };
+
+  return {
+    name: "octokit-bundles",
+    async setup(build) {
+      // Run the build steps after the main build
+      build.onEnd(async (result) => {
+        if (result.errors.length) return;
+
+        // Start with a clean slate
+        await fs.rm("pkg", { recursive: true, force: true });
```

Comment:
This should be moved out of the `build.onEnd()` function. It would delete the build folder after the build has completed